generators/messaging-adapter-generator/src/test/java/uk/gov/justice/raml/jms/it/JmsEndpointGenerationIT.java

```diff
@@ -83,8 +114,10 @@
             TestServiceContextNameProvider.class,
             DefaultJmsMessageLoggerHelper.class,
             DefaultTraceLogger.class,
-            DefaultJsonValidationLoggerHelper.class
+            DefaultJsonValidationLoggerHelper.class,
 
+            DefaultFileSystemUrlResolverStrategy.class,
+            GlobalValueProducer.class,
```

Comment:
Is this needed if we're using @Alternative?

Reply:
removed